### ISSUE TITLE
It is hoped that getFieldDecorator.options.trigger can collect child node data at multiple occasions.

### ISSUE THREAD

**Issue by MarxZhou** [ant-design/ant-design#25888]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I am using the color picker component react-color recommended by the official website.
I wrap it in the Form component.
This component has multiple APIs for changing the value of child nodes, such as onChange, onChangeComplete.
The configuration item trigger can only choose one of the two.
-------------
As a result, I will receive a warning in the console, such as the following message:
Warning: `getFieldDecorator` will override `color`, so please don't set `color` directly and use `setFieldsValue` to set it. 

### What does the proposed API look like?
I hope the trigger attribute can support string arrays to avoid this situation.
In other words, if the component has multiple methods for collecting the value of the child nodes, they can all be processed normally.

![image](https://user-images.githubusercontent.com/39861054/88786968-c1e51600-d1c5-11ea-9134-f4f6e40dca5a.png)

**Comment by afc163** [comment]
My suggestion is wrapping react-color to a custom Component, and unify `onChange` and `onChangeComplete` to one `onChange` callback.
